### ISSUE TITLE
shaper play lead to an error AttributeError: 'NoneType' object has no attribute 'items' during running shaper write

### ISSUE THREAD

**Issue by wenditer** [arno49/shaper#25]
My template contains following:
```
...
  test
    location
      TestComponent.properties:
        {% if some.property.var %}someProperty: '{{ some.property.var }}'{% endif %}
```

This produces following output after running shaper play:
```
  test
    location
      TestComponent.properties: null
```

I've generated templates.yaml from playbook which contains example of the code above. When I'm trying to run shaper write templates.yaml, it throws an error:
```
Traceback (most recent call last):
  File "d:\Env\Python\Python37\Scripts\shaper-script.py", line 11, in <module>
    load_entry_point('shaper', 'console_scripts', 'shaper')()
  File "c:\env\pes-v2-atg\build\workdir\shaper\shaper-master\shaper\cli.py", line 152, in main
    manager.write_properties(datastructure, arguments.out)
  File "c:\env\pes-v2-atg\build\workdir\shaper\shaper-master\shaper\manager.py", line 56, in write_properties
    os.path.join(directories, property_file),
  File "c:\env\pes-v2-atg\build\workdir\shaper\shaper-master\shaper\libs\parser.py", line 101, in write
    parser_class().write(data, path)
  File "c:\env\pes-v2-atg\build\workdir\shaper\shaper-master\shaper\libs\parser.py", line 302, in write
    '{}={}'.format(item[0], item[1]) for item in data.items()
AttributeError: 'NoneType' object has no attribute 'items'
```

**Comment by arno49** [comment]
fixed with 0.0.10

**Comment by wenditer** [comment]
still reproduced in 0.1.0
